src/TikzPictures.jl

```diff
@@ -258,25 +296,34 @@ end
 function save(f::SVG, tp::TikzPicture)
     try
         filename = f.filename
+        folder = abspath(".")
+        tempdir = mktempdir(folder)
+        pwd = abspath(".")
+        cd(abspath(tempdir))
         if tp.usePDF2SVG
             save(PDF(filename), tp)
-            success(`pdf2svg $filename.pdf $filename.svg`) || error("pdf2svg failure")
+            cd(pwd)
+            success(`pdf2svg $(tempdir * "/" * filename).pdf $filename.svg`) || error("pdf2svg failure")
             if tikzDeleteIntermediate()
-                rm("$filename.pdf")
+                # delete tmp dir
+                rm(tempdir, recursive=true)
             end
         else
-            save(TEX("$filename.tex"), tp)
+            save(TEX("$(filename).tex"), tp)
             if tp.enableWrite18
-                success(`$(tikzCommand()) --enable-write18 --output-format=dvi $filename`)
+                success(`$(tikzCommand()) --enable-write18 --output-format=dvi $(".")`)
```

Comment:
Why the `$(".")`?

Reply:
All the intermediate files are being created in the temp directory, but the tikzCommand() only works when in the same directory as the .tex file. It's messy, but I changed the working directory to the temp directory temporarily and executed the command then switched back.

Reply:
Why not just make `$(".")` just `.`?

Reply:
I was just replacing the text inside the parens. I'll change it now.